waiter/src/waiter/schema.clj

```diff
@@ -105,6 +105,16 @@
   [{:keys [kind] :as config}]
   (nil? (s/check {(s/required-key :factory-fn) s/Symbol, s/Keyword s/Any} (get config kind))))
 
+(defn valid-reverse-proxy-config
```

Comment:
This function should return the schema and we can invoke the `check` as a precondition in `kubernetes-scheduler`

Reply:
Done